tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "extends": "@stoplight/scripts/tsconfig.json",
+
   // target all ts files
-  "include": ["src"]
+  "include": ["."],
```

Comment:
We may need to reflect that change `package.json` as the structure of `dist` directory is likely to change, so I guess we may need `typings` field needs to be added?